libraries/provider_database_postgresql_user.rb

```diff
@@ -88,6 +88,14 @@ def action_grant_schema
         end
 
         private
+        def replication_keyword?
```

Comment:
I think this method should be refactored as def keyword?(word) and moved into the base provider.